Update virtualenv-safe-config.md (#982)
yandex-cloud/docs: en/_includes/security/standard/virtualenv-safe-config.md

### {{ objstorage-full-name }} {#objstorage}

#### 3.8 No public access to the {{ objstorage-name }} bucket {#bucket-access}

> It is recommended to assign minimum roles for a bucket using {{ iam-short-name }} and supplementing or itemizing them using a bucket policy (for example, to restrict access to the bucket by IP, grant granular permissions for objects, and so on).

Reviewer: ```suggestion
We recommend assigning minimum roles for a bucket using {{ iam-short-name }} and supplementing or itemizing them using a bucket policy (for example, to restrict access to the bucket by IP, grant granular permissions for objects, and so on).
```

1. Open the settings of a specific VM with a **Container Optimized Image**.
1. In the Docker container's Settings, disable the **Privileged mode** parameter.

> #### 3.31 {{ certificate-manager-full-name }} certificate is valid for at least 30 days {#certificate-validity}

Reviewer: ```suggestion
#### 3.31 The {{ certificate-manager-full-name }} certificate is valid for at least 30 days {#certificate-validity}
```

You can use {{ certificate-manager-full-name }} to manage TLS certificates for your API gateways in the API Gateway, as well as your websites and buckets in {{ objstorage-name }}. {{ alb-name }} is integrated with {{ certificate-manager-short-name }} for storing and installing certificates. it is recommended that you use {{ certificate-manager-short-name }} to obtain your certificates and rotate them automatically.

> When using TLS in your application, it is recommended that you limit the list of your trusted root certificate authorities (root CA).

Reviewer: ```suggestion
When using TLS in your application, we recommend that you limit the list of your trusted root certificate authorities (root CA).
```

When using certificate pinning, keep in mind that Let's Encrypt certificates are [valid for 90 days](https://letsencrypt.org/docs/faq/#what-is-the-lifetime-for-let-s-encrypt-certificates-for-how-long-are-they-valid).

> it is recommended that you update certificates in advance if they are not [updated automatically](../../../certificate-manager/concepts/challenges.md#auto).

Reviewer: ```suggestion
We recommend that you update certificates in advance if they are not [updated automatically](../../../certificate-manager/concepts/challenges.md#auto).
```

#### 3.31 The {{ certificate-manager-full-name }} certificate is valid for at least 30 days {#certificate-validity}

> You can use {{ certificate-manager-full-name }} to manage TLS certificates for your API gateways in the API Gateway, as well as your websites and buckets in {{ objstorage-name }}. {{ alb-name }} is integrated with {{ certificate-manager-short-name }} for storing and installing certificates. it is recommended that you use {{ certificate-manager-short-name }} to obtain your certificates and rotate them automatically.

Reviewer: ```suggestion
You can use {{ certificate-manager-full-name }} to manage TLS certificates for your API gateways in the API Gateway, as well as your websites and buckets in {{ objstorage-name }}. {{ alb-name }} is integrated with {{ certificate-manager-short-name }} for storing and installing certificates. We recommend that you use {{ certificate-manager-short-name }} to obtain your certificates and rotate them automatically.
```

**Guides and solutions to use:**

> Update the certificate or setup auto updates.

Reviewer: ```suggestion
Update the certificate or set up auto updates.
```

#### 3.36 Vulnerability scanning is performed at the cloud IP level {#ip-level}

> it is recommended that customers should scan their hosts for vulnerabilities by themselves. Cloud resources support the installation of custom virtual images of vulnerability scanners or software agents on hosts. There are many paid and free scanning solutions on the market.

Reviewer: ```suggestion
We recommend that customers should scan their hosts for vulnerabilities by themselves. Cloud resources support the installation of custom virtual images of vulnerability scanners or software agents on hosts. There are many paid and free scanning solutions on the market.
```

You can manage web sockets using the [API](../../../api-gateway/api-ref/websocket/authentication.md) that receives information about a connection, sends data to the client side, and closes the connection.

> it is recommended that you use the following when connecting to the API gateway via WebSocket:

Reviewer: ```suggestion
We recommend that you use the following when connecting to the API gateway via WebSocket:
```

#### 3.46 Authorization in the API gateway is configured {#authorization}

> it is recommended using the OpenAPI 3.0 authentication and authorization mechanisms that are standard for {{ api-gw-name }}. Currently, you can use authorization via a function and via a JWT.

Reviewer: We recommend using the OpenAPI 3.0 authentication and authorization mechanisms that are standard for {{ api-gw-name }}. Currently, you can use authorization via a function and via a JWT.
```
